src/PantheonTerminalWindow.vala

```diff
@@ -53,9 +39,19 @@ namespace PantheonTerminal {
         private const string SOLARIZED_LIGHT_BG = "rgba(253, 246, 227, 0.95)";
         private const string SOLARIZED_LIGHT_FG = "#586e75";
 
-
-
+        public bool unsafe_ignored;
+        public bool focus_restored_tabs { get; construct; default = true; }
+        public bool recreate_tabs { get; construct; default = true; }
+        public bool restore_pos { get; construct; default = true; }
+        public PantheonTerminalApp app { get; construct; }
```

Comment:
This duplicates Widget.application - is it advantageous to have a separate property as well as set_application (app) in construct {}?

Reply:
I'd like to chase this down in a follow-up PR, but there seems to be some conflict between `Gtk.Application` and `PantheonTerminalApp`. Trying to use `Widget.Application` is giving me errors about setting shortcut accelerators and some other issues.